src/CommonLib/LDAPUtils.cs

```diff
@@ -1498,7 +1525,6 @@ private string ResolveDomainToFullName(string domain)
         /// <param name="authType">Auth type to use. Defaults to Kerberos. Use Negotiate for netonly/cross trust(forest) scenarios</param>
         /// <param name="globalCatalog">Use global catalog or not</param>
         /// <returns>A connected LDAP connection or null</returns>
-
         private async Task<LdapConnectionWrapper> CreateLDAPConnectionWrapper(string domainName = null, bool skipCache = false,
```

Comment:
`CreateLDAPConnectionWrapper` is still a bit of a monolith, I'll want to iterate this one again.